tests/test_backup.py

```diff
@@ -110,3 +112,122 @@ def test_backup_vault_kms_filter(self):
         self.assertTrue(len(resources), 1)
         aliases = kms.list_aliases(KeyId=resources[0]['EncryptionKeyArn'])
         self.assertEqual(aliases['Aliases'][0]['AliasName'], 'alias/aws/backup')
+
+    def test_backup_vault_modify_add_remove_statements(self):
+        session_factory = self.replay_flight_data(
+            "test_backup_vault_modify_add_remove_statements"
+        )
+        client = session_factory().client("backup")
+        backup_vault = client.create_backup_vault(
+            BackupVaultName="test_backup_vault_modify_add_remove_statements"
+        )
+
+        backup_vault_name = backup_vault["BackupVaultName"]
+        backup_vault_arn = backup_vault["BackupVaultArn"]
+
+        client.put_backup_vault_access_policy(
```

Comment:
we don't really want all the resource creation/setup in code (re create, set policy), we're trying to move this stuff to terraform to avoid the bloat in recording and test code.